builtin/logical/transit/path_datakey.go

```diff
@@ -28,6 +28,12 @@ func (b *backend) pathDatakey() *framework.Path {
 ciphertext; "wrapped" will return the ciphertext only.`,
 			},
 
+			"padding_scheme": {
+				Type: framework.TypeString,
+				Description: `The padding scheme to use for decrypt. Currently only applies to RSA key types.
+Options are 'oaep' or 'pkcs1v15'. Defaults to 'oaep'`,
+			},
```

Comment:
It'd be great to include `Default: "oaep"` here for consistency if we could :-) When you call `d.Get(...)` below, it'll populate it with `oaep`.

Reply:
This applies in a few places as well below :-)

Reply:
If I were to nitpick, for consistency with [sign/verify I might suggest calling this `encryption_algorithm`](https://github.com/hashicorp/vault/blob/main/builtin/logical/transit/path_sign_verify.go#L150-L154), as it'll likely be more widely used than just RSA here. `padding_scheme` is definitely more precise, but boxes us into a corner w.r.t. EC algorithms.

Reply:
> I might suggest calling this `encryption_algorithm`

I think too, this makes a lot of sense and is consistent with the existing API.